flask-api/kujira.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python
+
+import argparse
+import json
+
+from AuthenticatedHttpClient import AuthenticatedHttpClient
+from flask import Flask
+
+app = Flask(__name__)
```

Comment:
Do not write everything in one file.
kujira/controllers/osds.py
kujira/controllers/servers.py